seed/views/v3/portfolio_manager.py

```diff
@@ -182,7 +182,7 @@ def report(self, request):
                     status=status.HTTP_400_BAD_REQUEST)
             try:
                 possible_properties = content_object['report']['informationAndMetrics']['row']
-                if isinstance(possible_properties, list):
+                if isinstance(possible_properties, list) or isinstance(possible_properties, dict):
                     properties = possible_properties
```

Comment:
Did you try something like this?
```
if it's a list:
  properties = possible_properties
else if it's a dict:
  properties = [possible_properties]
```
That might be nice so that down the line (ie in the view create_from_pm_import) we don't have to handle dicts and it would the api simpler

Reply:
That's genius thanks!